main.js

```diff
@@ -1,30 +1,37 @@
-var sidesRecipes = ['carrot sticks', 'broccoli', 'peppers & onions', 'roti', 'dumplings'];
-var mainDishRecipes = ['vegan burger', 'grilled tofu', 'channa masala', 'tofu mutar masala'];
-var dessertRecipes = ['brownies', 'ice cream', 'Tonys Chocolonely', 'cookies', 'popsicles'];
-var favRecipes = ['pizza', 'muffins', 'ice cream'];
+var sidesRecipes = ['Carrot Sticks', 'Broccoli', 'Peppers & Onions', 'Roti', 'Dumplings', 'Mashed Potatoes', 'Corn on the Cob', 'Salad', 'Chips & Salsa', 'Black Beans & Rice', 'Smashed Potatoes', 'Baked Potatoes', 'Grilled Asparagus'];
+var mainDishRecipes = ['Pesto, Tomato & Mozzarella Veggie Burger', 'BBQ Grilled Tofu', 'Channa Masala', 'Tofu Mutar Masala', 'Tofu Fried Rice', 'Green Curry with Tofu', 'Red Curry with Tofu', 'Spicy Peanut Tofu', 'Crispy Tofu with Honey Mustard', 'Nachos with Black Beans & Guacamole'];
+var dessertRecipes = ['Apple Pie', 'Lemon Meringue Pie', 'Black Forest Cake', 'Banana Bread', 'Peach Cobbler', 'Cheesecake', 'Funfetti Cake', 'Baklava', 'Flan', 'Macarons', 'Macaroons', 'Chocolate Cupcakes', 'Pavlova', 'Pumpkin Pie', 'Key Lime Pie', 'Tart Tatin', 'Croissants', 'Eclairs'];
+var favRecipes = ['pizza', 'chocolate', 'sushi'];
 var currentRecipe = "";
 
 var viewFavsButton = document.querySelector('#view-favs-button');
 var letsCookButton = document.querySelector('#lets-cook-button');
 var letsCookBox = document.querySelector('#lets-cook-box');
 
-var sideOption = document.querySelector('#side-option').value;
-var mainDishOption = document.querySelector('#main-dish-option').value;
-var dessertOption = document.querySelector('#dessert-option').value;
-var entireMealOption = document.querySelector('#entire-meal-option').value;
+var sideOption = document.querySelector('#side-option');
```

Comment:
Used the .checked method to get a truthy/falsey response to determine which array to output a recipe from.